### PR TITLE
Add the possibility of ignoring specified Tenor images

### LINKED ISSUE #139: Add the possibility of ignoring specified Tenor images
I suggest we will add a parameter called `TENOR_BLACKLIST` for a comma-separated list of the images which must be ignored. By default, the value of the parameter must be [641ee5344bdc3f9f4d3ef52344dfe6bd](https://media.tenor.com/images/641ee5344bdc3f9f4d3ef52344dfe6bd/tenor.gif). The point is that the picture has the tags `Harry` and `Potter`, but it's absolute rubbish.

### PR DESCRIPTION
**Pull request by ya7on**
close #139 
# Please make sure all below checkboxes in the Mandatory section have been checked before submitting your PR (also don't forget to pay attention to the Depending on Circumstances section)

## Mandatory

- [x] The commit message is an imperative and starts with a capital letter (for example, `Add something`, `Remove something`, `Fix something`, etc.).
- [x] I made sure I hadn't put a period at the end of the the commit message(s).

## Depending on Circumstances

Some of the items in the section become mandatory if you are a first-time contributor and/or your changes are relevant to the items.

- [x] Adding a new parameter I didn't forget to reflect it in `README.md`.
- [ ] Modifying a list of constants or imports I didn't brake an alphabetical order.
- [ ] (for first-time contributors) One of the commits in the pull request adds me to the **Acknowledgements** section in `AUTHORS.md`.

### CODE REVIEW

**Review by polina-popova — COMMENTED** [review]
Add new variable to the configuration in README.
for example: `A comma separated list of GIFs id which will be excluded from Tenor's respond to the search query.`

File: src/birthder.js [hunk 1]
```diff
@@ -128,10 +129,30 @@
...
+    const filteredItems = items.filter(item => {
+      const re = /^(https?[:]\/\/media.tenor.com\/images\/([0-9a-z]+)\/tenor.gif)$/i
+      const match = item.media[0].gif.url.match(re)
+
+      if (!match) {
+        return false
+      }
+
+      if (TENOR_BLACKLIST.includes(match[2])) {
+        console.log('Tenor returned the image from blacklist. Retrying...')
```
Comment (eugulixes) [inline src/birthder.js]: Let's amend the record a bit.
`Tenor returned the image from the blacklist specified via TENOR_BLACKLIST. Retrying...`

File: README.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ The script can be configured via the following environment variables (called par
 | Parameter                           | Description | Default |
 |-------------------------------------|-------------|---------|
 | `TENOR_API_KEY`                     | Сlient key for privileged API access. This is the only **mandatory** parameter. | |
+| `TENOR_BLACKLIST` | A comma separated list of GIFs id which will be excluded from Tenor's respond to the search query | [641ee5344bdc3f9f4d3ef52344dfe6bd](https://media.tenor.com/images/641ee5344bdc3f9f4d3ef52344dfe6bd/tenor.gif) |
```
Comment (eugulixes) [inline README.md]: Let's expand the current description.
```
A comma separated list of the GIFs ids which will be excluded when choosing one from the list returned by Tenor. If the script randomly choose a GIF from the blacklist, the script sends one more request to choose a different GIF.
```

**Review by eugulixes — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -45,6 +45,7 @@ The script can be configured via the following environment variables (called par
 | Parameter                           | Description | Default |
 |-------------------------------------|-------------|---------|
 | `TENOR_API_KEY`                     | Сlient key for privileged API access. This is the only **mandatory** parameter. | |
+| `TENOR_BLACKLIST` | A comma separated list of the GIFs ids which will be excluded when choosing one from the list returned by Tenor. If the script randomly chooses a GIF from the blacklist, the script sends one more request to choose a different GIF. | [641ee5344bdc3f9f4d3ef52344dfe6bd](https://media.tenor.com/images/641ee5344bdc3f9f4d3ef52344dfe6bd/tenor.gif) |
```
Comment (polina-popova) [inline README.md]: If the script randomly chooses a GIF from the response which belongs to the blacklist, the script sends one more request to Tenor.

**Review by eugulixes — APPROVED** [review]

**Pull request merged**